docs/source/interactive/python-ipython-diff.rst

```diff
@@ -58,8 +58,8 @@ All the following construct are valid IPython syntax:
        ...: print $months[0];
 
 
-Each of these construct is compiled by IPython into valid python code and will
-do most of the time what you expect it will do. Let see each of these example
+Each of these constructs is compiled by IPython into valid python code and will
+do most of the time what you expect it will do. Let see each of these examples
```

Comment:
"Let see" seems invalid too?

```suggestion
do most of the time what you expect it will do. Let's see each of these examples
```

Reply:
Good catch, thanks for the review. I'll update the PR with all/most of your suggestions included in a few minutes